app/queries/my_facilities/missed_visits_query.rb

```diff
@@ -95,14 +93,12 @@ def missed_visit_totals
 
   def visits_in_quarter(year, quarter, patients)
     LatestBloodPressuresPerPatientPerQuarter
-      .where(year: year, quarter: quarter)
-      .where(patient_id: patients)
+      .where(year: year, quarter: quarter, patient: patients)
```

Comment:
This clause used to be `patient_id: patients`, and now it is `patient: patients`. Can we confirm that both work?

Reply:
Yup, this works because `LatestBloodPressuresPerPatientPerQuarter` and `LatestBloodPressuresPerPatientPerMonth` both have a `belongs_to :patient` relationship. This needed to use `patient_id` earlier because the `patients` arg we were passing in was actually the result of the join up top, and not just a `Patient` relation.

Reply:
Perfect 👌